src/pixel.js

```diff
@@ -67,10 +70,25 @@ function createImagePixel(win, src, noReferrer = false, attributionSrc) {
   if (noReferrer) {
     image.referrerPolicy = 'no-referrer';
   }
-  image.src = src;
-  if (isAttributionReportingAllowed(win.document)) {
-    image.attributionsrc = attributionSrc;
+
+  let attributionReportingStatus =
+    AttributionReportingStatus.ATTRIBUTION_DATA_UNSPECIFIED;
+  if (attributionSrc != null) {
+    if (isAttributionReportingAllowed(win.document)) {
+      attributionReportingStatus =
+        AttributionReportingStatus.ATTRIBUTION_DATA_PRESENT_AND_POLICY_ENABLED;
+      attributionSrc = attributionSrc.replace(
```

Comment:
Need to use urlReplacementService to do argument substitution: https://github.com/ampproject/amphtml/blob/3c58857f178d2b81a62304d61ab0419cf00ba569/extensions/amp-ad-exit/0.1/amp-ad-exit.js#L208

Using `replace` would not be safe.

Reply:
Updated, thanks.